lib/internal/modules/esm/resolve.js

```diff
@@ -117,17 +94,9 @@ function getPackageConfig(path, base) {
   let packageJSON;
   try {
     packageJSON = JSONParse(source);
-  } catch {
-    const packageConfig = {
-      exists: true,
-      main: undefined,
-      name: undefined,
-      isValid: false,
-      type: 'none',
-      exports: undefined
-    };
-    packageJSONCache.set(path, packageConfig);
-    return packageConfig;
+  } catch (error) {
+    const errorPath = StringPrototypeSlice(path, 0, path.length - 13);
```

Comment:
what is this magic number 13 for?

Reply:
I guess `'package.json'.length - 1`

Reply:
a comment to that effect may help then :-)